api/docker/php/docker-entrypoint.sh

```diff
@@ -11,56 +11,36 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ] || [ "$1
 	fi
-	#ln -sf "$PHP_INI_RECOMMENDED" "$PHP_INI_DIR/php.ini"
 
 	mkdir -p var/cache var/log
-	#setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX var
-	#setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX var
 
-	if [ "$APP_ENV" != 'prod' ] && [ ! -f config/jwt/private.pem ]; then
-		jwt_passphrase=${JWT_PASSPHRASE:-$(grep ''^JWT_PASSPHRASE='' .env | cut -f 2 -d ''='')}
-		if ! echo "$jwt_passphrase" | openssl pkey -in config/jwt/private.pem -passin stdin -noout > /dev/null 2>&1; then
-			echo "Generating public / private keys for JWT"
-			mkdir -p config/jwt
-			echo "$jwt_passphrase" | openssl genpkey -out config/jwt/private.pem -pass stdin -aes256 -algorithm rsa -pkeyopt rsa_keygen_bits:4096
-			echo "$jwt_passphrase" | openssl pkey -in config/jwt/private.pem -passin stdin -out config/jwt/public.pem -pubout
-			setfacl -R -m u:www-data:rX -m u:"$(whoami)":rwX config/jwt
-			setfacl -dR -m u:www-data:rX -m u:"$(whoami)":rwX config/jwt
-		fi
+	if [ "$APP_ENV" = 'prod' ]; then
+	  ln -sf "$PHP_INI_RECOMMENDED" "$PHP_INI_DIR/php.ini"
+	  setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX var
+	  setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX var
 	fi
 
 	if [ "$APP_ENV" != 'prod' ]; then
-		composer install --prefer-dist --no-progress --no-interaction
-	fi
-
-	if grep -q DATABASE_URL= .env; then
-		echo "Waiting for database to be ready..."
-		ATTEMPTS_LEFT_TO_REACH_DATABASE=60
-		until [ $ATTEMPTS_LEFT_TO_REACH_DATABASE -eq 0 ] || DATABASE_ERROR=$(php bin/console dbal:run-sql -q "SELECT 1" 2>&1); do
-			if [ $? -eq 255 ]; then
-				# If the Doctrine command exits with 255, an unrecoverable error occurred
-				ATTEMPTS_LEFT_TO_REACH_DATABASE=0
-				break
-			fi
-			sleep 1
-			ATTEMPTS_LEFT_TO_REACH_DATABASE=$((ATTEMPTS_LEFT_TO_REACH_DATABASE - 1))
-			echo "Still waiting for database to be ready... Or maybe the database is not reachable. $ATTEMPTS_LEFT_TO_REACH_DATABASE attempts left."
-		done
+    if [ ! -f config/jwt/private.pem ]; then
+      jwt_passphrase=${JWT_PASSPHRASE:-$(grep ''^JWT_PASSPHRASE='' .env | cut -f 2 -d ''='')}
+      if ! echo "$jwt_passphrase" | openssl pkey -in config/jwt/private.pem -passin stdin -noout > /dev/null 2>&1; then
+        echo "Generating public / private keys for JWT"
+        mkdir -p config/jwt
+        echo "$jwt_passphrase" | openssl genpkey -out config/jwt/private.pem -pass stdin -aes256 -algorithm rsa -pkeyopt rsa_keygen_bits:4096
+        echo "$jwt_passphrase" | openssl pkey -in config/jwt/private.pem -passin stdin -out config/jwt/public.pem -pubout
+        setfacl -R -m u:www-data:rX -m u:"$(whoami)":rwX config/jwt
+        setfacl -dR -m u:www-data:rX -m u:"$(whoami)":rwX config/jwt
+      fi
+    fi
 
-		if [ $ATTEMPTS_LEFT_TO_REACH_DATABASE -eq 0 ]; then
-			echo "The database is not up or not reachable:"
-			echo "$DATABASE_ERROR"
-			exit 1
-		else
-			echo "The database is now ready and reachable"
-		fi
+		composer install --prefer-dist --no-progress --no-interaction
```

Comment:
This runs composer whenever a pod is restarting, right? Is this needed?

All dependencies should already be included in the docker image and this would only slow down pod starts / scaling.

Reply:
This as well is inside a big `if [ "$APP_ENV" != 'prod' ]`, so it does not run on the deployments.

Reply:
Ah, I didn't recognize the `if [ "$APP_ENV" != 'prod' ]` above. Make sense to me.

Reply:
> That is indeed a valid attack vector. I guess we need to be extra careful when deploying a PR from an unknown collaborator.

Ok, I see. So it makes extra sense to not share any secrets between PR/dev environment and stage/prod environment, so in worst case, an attacker would only have access to dev secrets.

Reply:
By the way: Would be good if you can fix the indentation. It's now mixed tabs and spaces which makes it hard to read, both here on Github but also in editor (just opened the file locally, as well).